### ISSUE TITLE
Incorrect model serving with `--standardize False`

### ISSUE THREAD

**Issue by aguschin** [iterative/mlem#593]
After #588 it appears that models are sometimes served incorrectly:

```py
def hash_text(text, salt="SALT"):
    return hash(text + salt)


from mlem.api import save

save(hash_text, "hash_model", sample_data="sometext")
```
then
```shell
$ mlem serve streamlit -m hash_model --standardize False
⏳️ Loading model from hash_model.mlem
Starting streamlit server...
🖇️  Adding route for /__call__
Checkout openapi docs at <http://0.0.0.0:8080/docs>
...
```
and I see
<img width="778" alt="image" src="https://user-images.githubusercontent.com/6797716/214508771-d746e338-8e9c-42fc-b524-8d7ad7fb934d.png">
the problem: it's the wrong input to be expected from the model. It should be `text`. If you input something and click on "Submit", it fails.
In FastAPI:
<img width="381" alt="image" src="https://user-images.githubusercontent.com/6797716/214508984-8e46356e-05c9-467c-9956-23fd5ab33abd.png">

Interestingly, it works with `--standardize True`:
<img width="804" alt="image" src="https://user-images.githubusercontent.com/6797716/214509208-a9bb631e-2ceb-461b-977c-8fced570f7f1.png">

**Comment by mike0sv** [comment]
This should be fixed by #595, please check

**Comment by aguschin** [comment]
yep! thanks!
